Classes/common/touchdb/ChangeTracker/TDURLConnectionChangeTracker.m

```diff
@@ -103,9 +110,10 @@ - (BOOL)start
 
 - (void)clearConnection
 {
-    [self.connection cancel];
-    [self.connection unscheduleFromRunLoop:[NSRunLoop currentRunLoop] forMode:NSRunLoopCommonModes];
-    self.connection = nil;
+    if(self.task.state != NSURLSessionTaskStateCompleted){
```

Comment:
I don't think so (doc [here](https://developer.apple.com/library/ios/documentation/Foundation/Reference/NSURLSessionTask_class/index.html#//apple_ref/doc/c_ref/NSURLSessionTaskState)). I guess we could also check for cancelling, but I'm not sure if that gains us anything